docs/reference/asyncapi-rules.md

```diff
@@ -0,0 +1,311 @@
+# AsyncAPI Rules
+
+Spectral has a built-in "asyncapi" ruleset for the [AsyncAPI Specification](https://www.asyncapi.com/docs/specifications/2.0.0/).
+
+In your ruleset file you can add `extends: "spectral:asyncapi"` and you'll get all of the following rules applied.
+
+## AsyncAPI v2-only
+
+These rules will only apply to AsyncAPI v2 documents.
+
+### asyncapi2-channel-no-empty-parameter
+
+Channel parameter declarations cannot be empty, ex.`/given/{}` is invalid.
+
+**Recommended:** Yes
+
+### asyncapi2-channel-no-query-nor-fragment
+
+Query parameters and fragments shouldn't be used in channel names. Instead, use bindings to define them.
+
+**Recommended:** Yes
+
+### asyncapi2-channel-no-trailing-slash
+
+Keep trailing slashes off of channel names, as it can cause some confusion. Most messaging protocols will treat `example/foo` and `example/foo/` as different things. Keep in mind that tooling may replace slashes (`/`) with protocol-specific notation (e.g., `.` for AMQP), therefore, a trailing slash may result in an invalid channel name in some protocols.
```

Comment:
```suggestion
Keep trailing slashes off of channel names, as it can cause some confusion. Most messaging protocols will treat `example/foo` and `example/foo/` as different things. Keep in mind that tooling may replace slashes (`/`) with protocol-specific notation (e.g.: `.` for AMQP), therefore, a trailing slash may result in an invalid channel name in some protocols.
```

Reply:
@philsturgeon Fixed

Reply:
@philsturgeon Fixed